skimage/util/process.py

```diff
@@ -0,0 +1,68 @@
+from multiprocessing import cpu_count
+
+import dask.array as da
+
+__all__ = ['process_chunks']
+
+
+def _get_chunks(shape, ncpu):
```

Comment:
We could also have the get_chunks function in skimage just return a single tuple of blocksizes.  This is probably more intuitive for users and dask.array functions will convert to the more explicit tuple of tuples form automatically.